android/PhysicalWeb/app/src/main/res/values/strings.xml

```diff
@@ -19,12 +19,18 @@
     <string name="title_edit_urls">Edit URLs</string>
     <string name="title_nearby_beacons">Nearby Beacons</string>
     <string name="title_about">About</string>
+    <string name="title_settings">Settings</string>
     <string name="empty_nearby_beacons_list_text">Searching for beacons</string>
     <string name="about_app_title">Physical Web</string>
     <string name="url_getting_started">        https://google.github.io/physical-web/mobile/android/getting-started.html</string>
     <string name="config_no_beacons_found">No beacons found</string>
     <string name="notification_view_more_beacons">VIEW MORE</string>
     <string name="config_url_read_error">Error Reading URL</string>
+    <string name="pws_endpoint_setting_display">Physical Web Service</string>
+    <string name="pws_endpoint_setting_key">pws_endpoint</string>
+    <string name="google_pws">Google\'s PWS</string>
```

Comment:
I'm thinking we should use these strings:
* Google Physical Web Service
* Production Open Source Physical Web Service
* Development Open Source Physical Web Service

Start with that, and maybe @scottjenson can comment.